Tests/ScenarioTester/ScenarioTester/Request.cs

```diff
@@ -107,10 +112,11 @@ public static string Get(string url, string bearerToken = null)
         /// <param name="content"><see cref="RequestContent"/> to send to the server</param>
         /// <param name="bearerToken">Optional bearer token to pass to the request Auth header</param>
         /// <param name="additionalHeaders">Any additional headers to add to the request</param>
+        /// <param name="sslTrust">Optional option to tell whether to do or not ssl validation</param>
```

Comment:
why would we skip ssl validation?

Reply:
The purpose is to allow us to make Scenario Tester to trust DataX server certificate without further configurations. When the ssl certificate is not properly configured, the client wont trust the api certificate and it would throw an exception. The idea of that flag is to make optional this validation from the client side, for development purposes, and make it configurable once the code reaches a production environment.

Reply:
The default is False now, skipping certificate validations needs to be explicitly defined in the properties for development purposes, otherwise, it will perform certificate validations as normal

Reply:
Default behavior will be to always verify certificate validation (dont skip validation). If skipping is needed, it needs to be set explicitly through configuration.